tests/test_parser.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from unittest.mock import MagicMock, patch
+from Bio import SeqIO
```

Comment:
Should we depend on Biopython you think? I'm always on the fence about this lol

Reply:
A FASTA parser is pretty trivial, so I have no opinion here

Reply:
Actually yes, I will write a FASTA parser, since we are already parsing and manipulating the FASTA identifier line

Reply:
Ok, let's leave it for now.